.github/workflows/jira-issue-transfer.yml

```diff
@@ -31,20 +31,26 @@ jobs:
             JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
             GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         - name: Jira Create Bug
-          if: github.event.label.name == 'ticketed' && !contains(github.event.issue.labels.*.name, 'ticketed') && contains(github.event.issue.labels.*.name, 'bug') 
+          id: bug
+          if: contains(github.event.issue.labels.*.name, 'ticketed', 'bug') && !contains(github.event.issue.labels.*.name, 'ticketed') 
```

Comment:
This logic seems contradictory? The first checks if it contains `ticketed` and the second makes sure it does _not_ contain `ticketed`.

Reply:
In addition, `contains()` only supports 2 parameters

https://docs.github.com/en/actions/learn-github-actions/expressions#contains